docs/manual/commands/interfaces.rst

```diff
@@ -0,0 +1,165 @@
+.. _scripting-interfaces:
+
+==========
+Interfaces
+==========
+
+Introduction
+============
+
+This page provides an overview of the various interfaces available to interact with Qtile's
+command graph.
+
+- ``lazy`` calls
+- when running ``qtile shell``
+- when running ``qtile cmd-obj``
+- when using ``CommandClient`` or ``InteractiveCommandClient`` in python 
```

Comment:
I suspect these are meant to be rendered as bullet points but they are not:
![image](https://user-images.githubusercontent.com/44536113/212436280-35972e49-20a5-44cb-b7bd-09e63a522e55.png)